x-pack/plugin/identity-provider/src/main/java/org/elasticsearch/xpack/idp/saml/authn/SamlAuthnRequestValidator.java

```diff
@@ -156,8 +155,8 @@ private void validateAuthnRequest(
             if (Strings.hasText(parsedQueryString.signature)) {
                 if (Strings.hasText(parsedQueryString.sigAlg) == false) {
                     logAndRespond(
-                        new ParameterizedMessage(
-                            "Query string [{}] contains a Signature but SigAlg parameter is missing",
+                        org.elasticsearch.core.Strings.format(
```

Comment:
We should look at either consolidating or renaming the two "Strings" classes in server/core, so that fqcn usages are not necessary.

Reply:
that would be great. There is more than two.. :)  I created an issue to track this. https://github.com/elastic/elasticsearch/issues/87610